client/components/domains/registrant-extra-info/fr-schema.json

```diff
@@ -1,5 +1,5 @@
 {
-	"id": "/contact-validation/fr",
+	"id": "https://public-api.wordpress.com/rest/v1/domains/validation-schema/fr",
```

Comment:
This URL doesn't seem to resolve to anything. Is there any reason to keep the `id`, or can it just be removed?

[`id` in the spec](https://tools.ietf.org/html/draft-zyp-json-schema-04#section-7.2)

Reply:
Ah-ha! #21524

Reply:
My reading of the spec is that the id URI doesn't have to resolve to anything, but I intend to have the schema available there sooner or later, so it seems a good choice.

We do need to have some `id` to resolve `refs` when we're consuming on the backend. You can check out the `TLD_Validator` if you're curious.

Reply:
Sounds good 👌 

Do you mind pulling these schema fixes into another PR? I'd like to keep this one focused.

Reply:
No worries - https://github.com/Automattic/wp-calypso/pull/21588

Reply:
That's deployed now, so you should be able to rebase it out of the history.

Reply:
Thanks!